### PR TITLE
M3-5589: Added Storybook Story for File Uploads

### PR DESCRIPTION
**Pull request by bnussman**
## Description

Adds a Storybook Story for our File Uploader

Because we don't have a generic file uploader component (😔), there are two variants of the File Uploader in this story
- Image Uploader
- Object Uploader

## How to test

- `yarn storybook`
- Find the `Components`/`File Uploader` story and test it!

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]
Looks good!

File: packages/manager/src/components/FileUploader/FileUploader.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
+import FileUploader from './FileUploader';
+import ObjectUploader from '/src/features/ObjectStorage/ObjectUploader/ObjectUploader';
+
+<Meta title="Components/File Uploads" />
```
Comment (tiffwong) [inline packages/manager/src/components/FileUploader/FileUploader.stories.mdx:5]: The documentation says "Uploader" but the side menu has these stories listed says "Uploads" so I think we should make them consistent.

Screen Shot 2022-02-14 at 4 08 52 PM

File: packages/manager/src/components/FileUploader/FileUploader.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      description: "My Ubuntu Image for Production",
+      dropzoneDisabled: false,
+      apiError: undefined,
+      setErrors: undefined,
+      setCancelFn: undefined,
+      onSuccess: undefined,
+    }}
+    argTypes={{
+      label: {
+        description: "The label of the upload that will be sent to the Linode API (used for Image uploads)",
```
Comment (tiffwong) [inline packages/manager/src/components/FileUploader/FileUploader.stories.mdx:41]: Also all the props here look like they're required other than `onSuccess` 🤔

**Review by HanaXu — CHANGES_REQUESTED** [review]
Setting `apiError` doesn't seem to do anything 🤔 . If we're keeping that field, can we initialize an empty string so that there's no extra `Set...` button click? And maybe set the controls for the function fields to null as well?

image

**Review by HanaXu — APPROVED** [review]

**Pull request merged**